scripts/start_civ.sh

```diff
@@ -26,12 +26,12 @@ GUEST_RPMB_DEV_SOCK=
 GUEST_THERMAL_DAEMON_PID=
 GUEST_BATTERY_DAEMON_PID=
 GUEST_IMAGE=$WORK_DIR/android.qcow2
-GUEST_VSOCK="-device vhost-vsock-pci,id=vhost-vsock-pci0,guest-cid=3"
+GUEST_VSOCK="-device vhost-vsock-pci,id=vhost-vsock-pci0,guest-cid=3,guest-cid=4"
 GUEST_SHARE_FOLDER=
 GUEST_SMBIOS_SERIAL=$(dmidecode -t 2 | grep -i serial | awk '{print $3}')
 GUEST_QMP_SOCK=$WORK_DIR/.civ.qmp.sock
 GUEST_QMP_UNIX="-qmp unix:$GUEST_QMP_SOCK,server,nowait"
-GUEST_NET="-device e1000,netdev=net0 -netdev user,id=net0,hostfwd=tcp::5555-:5555,hostfwd=tcp::5554-:5554"
+GUEST_NET="-device e1000,netdev=net0 -netdev user,id=net0,hostfwd=tcp::5555-:5555,hostfwd=tcp::5554-:5554,hostfwd=tcp::8085-:8085,hostfwd=tcp::1234-:1234""
```

Comment:
1. Why use port 8085 and 1234? sugguest not use 1234 since it is QEMU's default debug port for GDB. 
2. remove the redundent quota